lib/metadata/in_memory/backend.js

```diff
@@ -118,6 +166,29 @@ const metastore = {
                 if (!metadata.keyMaps.get(bucketName).has(objName)) {
                     return cb(errors.NoSuchKey);
                 }
+                if (params && params.versionId) {
+                    const baseKey = inc(formatVersionKey(objName, ''));
```

Comment:
@LaurenSpiegel please stop the demand for comments. A piece of code, unlike the specifications for a function or an API, speaks for itself and requires no comments for people to understand it, unless it is not self containing.